### PR TITLE
Fix build with Qt-5.11

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

File: avogadro/qtplugins/gamessinput/gamessinput.cpp [hunk 1]
```diff
@@ -24,9 +24,9 @@
 
 #include <molequeue/client/jobobject.h>
 
-#include <QtCore/QDebug>
-
-#include <QtWidgets/QMessageBox>
+#include <QAction>
+#include <QDebug>
```
Comment (cryos) [inline avogadro/qtplugins/gamessinput/gamessinput.cpp]: Does this require that we remove the Qt module, or just the addition of the QAction include?

Comment (a17r) [reply avogadro/qtplugins/gamessinput/gamessinput.cpp]: No, the fix is just adding the missing QAction header. But with Qt5 the recommendation is to not use the Qt module prefix anymore, so I drop wherever I see them.

Comment (cryos) [reply avogadro/qtplugins/gamessinput/gamessinput.cpp]: Fair enough, we use it elsewhere so I would prefer not to drop  it, I can append to your commit and merge that though.

Comment (a17r) [reply avogadro/qtplugins/gamessinput/gamessinput.cpp]: I can force-push just the QAction change in a second if you want me to

Comment (cryos) [reply avogadro/qtplugins/gamessinput/gamessinput.cpp]: That would be awesome if you don't mind

**Comment by cryos** [comment]
Merged. Thanks.

**Pull request merged**